src/tests/multihost/admultidomain/test_multidomain.py

```diff
@@ -138,12 +138,12 @@ def test_0002_bz2018432(multihost, newhostname, adjoin):
         domain_list_cmd = multihost.client[0].run_command(
             'sssctl domain-list', raiseonerr=False)
         domain_list = domain_list_cmd.stdout_text.split('\n')
-        domain_list.remove("implicit_files")
-        domain_list = domain_list[:-1]
```

Comment:
I am not sure why there is another element removed from the domain_list when implicit_files should be already gone.

Reply:
This test contains multiple forests, domain-list only returns the domains that are in the first forest. Removing the last element should remove the last domain in the other forest, but it is the wrong list.